app/public/js/profile_save_button.js

```diff
@@ -0,0 +1,20 @@
+$(document).ready(function() {
+	$("#profileForm").change(formChange);
+	$("#resetButton").click(resetButton);
```

Comment:
Remove the reset button and associated JavaScript.

I like how you used serialize.  I didn't know about it before and it works with less code than I thought would have been necessary.